include/SFML/System/Angle.inl

```diff
@@ -0,0 +1,259 @@
+////////////////////////////////////////////////////////////
+//
+// SFML - Simple and Fast Multimedia Library
+// Copyright (C) 2007-2022 Laurent Gomila (laurent@sfml-dev.org)
+//
+// This software is provided 'as-is', without any express or implied warranty.
+// In no event will the authors be held liable for any damages arising from the use of this software.
+//
+// Permission is granted to anyone to use this software for any purpose,
+// including commercial applications, and to alter it and redistribute it freely,
+// subject to the following restrictions:
+//
+// 1. The origin of this software must not be misrepresented;
+//    you must not claim that you wrote the original software.
+//    If you use this software in a product, an acknowledgment
+//    in the product documentation would be appreciated but is not required.
+//
+// 2. Altered source versions must be plainly marked as such,
+//    and must not be misrepresented as being the original software.
+//
+// 3. This notice may not be removed or altered from any source distribution.
+//
+////////////////////////////////////////////////////////////
+
+namespace priv
+{
+    constexpr float pi = 3.141592654f;
+    constexpr float tau = 2 * pi;
+
+    constexpr float positiveRemainder(float a, float b)
+    {
+        assert(b > 0.0f);
+        if (a >= 0.f)
+            return a - static_cast<float>(static_cast<int>(a / b)) * b;
+        else
+            return a + b - static_cast<float>(static_cast<int>(a / b)) * b;
+    }
+}
+
+
+////////////////////////////////////////////////////////////
+constexpr Angle::Angle() :
+m_radians(0.0f)
+{
+}
+
+
+////////////////////////////////////////////////////////////
+constexpr float Angle::asDegrees() const
+{
+    return m_radians * 180 / priv::pi;
```

Comment:
We might want to write `m_radians * (180.f / priv::pi)`, ensuring that we use a constant factor.
Avoiding the divison at runtime also mitigates concerns about conversions and loss of precision.

Same is of course true for `sf::degrees()`.

Reply:
Good catch.